tests/assets/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>ostrich example</title>
+
+    <style>
+        canvas {
+            border: 1px solid blacK;
+        }
+    </style>
+
+</head>
+
+<body>
+
+    <canvas id="canvas"></canvas>
+
+    <script src="../../dist/main.js"></script>
+    <script>
```

Comment:
The initialization of ostrich should now be part of the test-case as we obviously would like to test different settings in different cases.

Reply:
Will set this up.